enhancements/cluster-logging/elasticsearch-cert-management.md

```diff
@@ -143,11 +143,11 @@ We need to have a proper upgrade strategy from our LTS version to the version th
 
 ## Drawbacks
 
-None
+Given the OCP cert signing service does not allow for mTLS configuration with custom fields that the ES cluster requires, we are unable to use it so we have to implement something similar to a pre-existing service.
 
 ## Alternatives
 
-None
+Stay with the crrent implementation where any consumer of the Elasticsearch Operator is responsible for cert and secret management.
```

Comment:
I see now. If we don't use the annotation at all, we can just use our current strategy, right? We'll eventually move to this new approach, but there will be scenarios where the ES Operator will be at a newer version than the Jaeger, so, wanted to make sure they'll still work.

Reply:
not quite, "Alternatives" would be alternatives to the enhancement... not alternatives to opting in. This would still be a coordinated effort between Elasticsearch Operator, Cluster Logging Operator, and Jaeger Operator.

I can try to think of a way to somehow opt out for the Jaeger use case while they try to catch up... that may be what we need to do

Reply:
My concern is that we have currently 2 streams out there, 1.17 and 1.20, both of which currently provision ES clusters generating certs on their own. Just releasing an ES Operator that breaks those installations doesn't sound like a good plan: even if we decide to backport the required changes to those branches, we still risk having folks on an older Z version that is going to break.

To be clear: the main goal for this proposal is sound and I look forward to offloading the cert management from the Jaeger Operator to the ES Operator, but we do have a good installation base out there to take care of.

Reply:
Ack, 100% agree and understand. Let me figure out and add something about opting in (perhaps it can be an annotation on the elasticsearch CR) so we can preserve legacy systems and not introduce a huge break

Reply:
@jpkrohling added some new language to hopefully capture and address this concern. Please let me know if this makes sense.
@jcantrill please read though this too as it would change steps CLO needs to take (likely makes this easier)